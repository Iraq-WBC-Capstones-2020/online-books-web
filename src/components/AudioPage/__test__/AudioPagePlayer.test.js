import React from 'react';
import renderer from 'react-test-renderer';
import AudioPageSection from '../AudioPage';

test('AudioPagePlayer snapshot', () => {
  const tree = renderer.create(<AudioPageSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
