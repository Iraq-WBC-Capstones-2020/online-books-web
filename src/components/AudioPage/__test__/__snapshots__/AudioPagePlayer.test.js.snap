// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPagePlayer snapshot 1`] = `

  <div
      src={audio}
      className="height-200"
      customVolumeControls={[]}
      customAdditionalControls={[]}
      showFilledProgress={false}
    />
`;
